sdk/formrecognizer/ai-form-recognizer/dom-shims.d.ts

```diff
@@ -3,6 +3,17 @@
 
```

Comment:
So the shims.d.ts under samples was deleted.  `dom-shims.d.ts` was added to package root. Git thinks it's a rename.